tests/templates/kuttl/tls/untrusted-ca.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What is this cert? Where does it come from?

Reply:
Ìts generated from the smoke/certs/generate.sh script and has an unknown / untrusted CA.

Reply:
Can we add a note about this?

I'd also consider renaming it to `untrusted-cert.crt` or something if it's not an actual CA certificate.